docs/help-center/using-famedly/notes.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: Notes
+title: Notes
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Notes</h2>
+    <p class="hero__subtitle">Save important messages, media and even message yourself!</p>
+  </div>
+</div>
+
+## Add to Notes
+
+You can save messages and media from other conversations by adding it to <b>Notes</b>.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Move the cursor over the message you want to save.</li>
```

Comment:
I would suggest just to write "Click on the message you want to save" like we did that for all features in docs/help-center/using-famedly/messages.mdx